### PR TITLE
Add federated Authenticator Type to Spec

### PR DESCRIPTION
**Pull request by zhiruidu-okta**
Add missing federated type to authenticatorType spec

### CODE REVIEW

File: dist/spec.yaml [hunk 1]
```diff
@@ -10693,7 +10694,7 @@ definitions:
       x5t:
         readOnly: false
         type: string
-      'x5t#S256':
+      x5t#S256:
```
Comment (bretterer) [inline dist/spec.yaml:10697]: Is this change correct?

Comment (laura-rodriguez) [reply dist/spec.yaml:10697]: I think this change is generated automatically depending on your env.

**Review by laura-rodriguez — COMMENTED** [review]
@zhiruidu-okta the change looks pretty straightforward and it LGTM. However, I'm not seeing this new type listed in our [API Reference](https://developer.okta.com/docs/reference/api/authenticators-admin/#authenticator-object). Should we also update the docs or we wanna keep this type "internal"?

File: dist/spec.yaml (hunk 1 above)
Comment (zhiruidu-okta) [reply dist/spec.yaml:10697]: Yeah, happens after running yarn build. I assumed it was by design.

**Comment by zhiruidu-okta** [comment]
> @zhiruidu-okta the change looks pretty straightforward and it LGTM. However, I'm not seeing this new type listed in our [API Reference](https://developer.okta.com/docs/reference/api/authenticators-admin/#authenticator-object). Should we also update the docs or we wanna keep this type "internal"?

@laura-rodriguez Literally just made the change to the API docs if you check again ;p

**Review by laura-rodriguez — APPROVED** [review]
LGTM 👍

**Review by bretterer — APPROVED** [review]

**Pull request merged**